lektor/admin/static/js/widgets.jsx

```diff
@@ -58,13 +58,14 @@ FallbackWidget.propTypes = widgetPropTypes;
 /**
  * An input widget wrapped in a <div> with description and label.
  */
+
 export const FieldBox = React.memo(function FieldBox(props) {
   const { field, value, placeholder, disabled } = props;
   const onChange = props.onChange
     ? props.onChange
-    : (value) => props.setFieldValue(field, value);
+    : (value, uiChange) => props.setFieldValue(field, value, uiChange);
   const className = "col-md-" + getFieldColumns(field) + " field-box";
-  let innerClassName = "field";
+  let innerClassName = `field ${field.type.name}`;
```

Comment:
This change seems to be a bit unrelated, is it wanted/necessary?

Reply:
Sorry I have a massively hacked branch that is styling the edit box contents on field type. This just leaked.
It is unecessary for this fix.

Reply:
No worries, can you remove it then?

Reply:
Done